core/src/main/java/io/javaoperatorsdk/jenvtest/KubeAPIServer.java

```diff
@@ -12,7 +12,7 @@ public class KubeAPIServer implements UnexpectedProcessStopHandler {
 
   private static final Logger log = LoggerFactory.getLogger(KubeAPIServer.class);
 
-  public static final int STARTUP_TIMEOUT = 15_000;
+  public static final int STARTUP_TIMEOUT = 10_0000;
```

Comment:
100 seconds? That seems too much to wait.

Reply:
yeah that is a mistake